test/pytest.ini

```diff
@@ -1,8 +1,14 @@
 [pytest]
+required_plugins = pytest-env
 markers =
     slow: mark test as slow.
     service_restart: test restarts systemd service (this tests can not be executed in the container)
     ipa: test working with FreeIPA (client or server)
 filterwarnings =
     ignore::DeprecationWarning
     ignore:Unverified HTTPS request is being made to host.*::
+env =
```

Comment:
I don't get it, what's the purpose of using pytest-env here? It looks like this actually prevents passing values through environment variables.

Reply:
pytest-env is an extension for pytest through which we can specify env variables for tests in the pytest.ini file. Use-case for this: not to specify all these values as CLI options. And the point of having this variable here is: not to litter the system with these magic (from the rest of the system perspective) env variables, but to have them in place related to tests. So, from the user side, that is no need to add any `export` somewhere, but just to specify his own values in this place

Reply:
1. you already have `--ipa-ip`, why have a second layer of parametrization? if you prefer envvars, why keep `--ipa-ip`?
2. https://github.com/MobileDynasty/pytest-env/blob/master/README.md reads like it's for setting variables to something other than what the caller sets. if you don't use `D:` prefix, you override, but then if you were to use `D:` prefix, then why default to empty? all in all, I think pytest-env brings negative value over just `getenv`ing the values.